src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/adapters/TableAdapter.test.js

```diff
@@ -222,6 +222,61 @@ test('Render data with skin', () => {
     expect(tableAdapter).toMatchSnapshot();
 });
 
+test('Render data with shrunken cell', () => {
+    const data = [
+        {
+            id: 1,
+            status: 'planned',
+        },
+        {
+            id: 2,
+            status: 'running',
+        },
+        {
+            id: 3,
+            status: 'succeeded',
+        },
+        {
+            id: 4,
+            status: 'failed',
+        },
+    ];
+
+    const schema = {
+        title: {
+            filterType: null,
+            filterTypeParameters: null,
+            transformerTypeParameters: {},
+            type: 'string',
+            sortable: true,
+            visibility: 'no',
+            label: 'Title',
+            width: 'shrink',
+        },
+        description: {
+            filterType: null,
+            filterTypeParameters: null,
+            transformerTypeParameters: {},
+            type: 'string',
+            sortable: true,
+            visibility: 'yes',
+            label: 'Description',
+            width: 'shrink',
```

Comment:
think i would use `auto` and `shrink` in this test case

```suggestion
            width: 'auto',
```